packages/forma-36-website/src/pages/foundation/typography/design.mdx

```diff
@@ -0,0 +1,128 @@
+import {
+  Card,
+  Typography,
+  Paragraph,
+  Heading,
+  Subheading,
+  SectionHeading,
+  DisplayText
+} from '@contentful/forma-36-react-components';
+
+
+Typography is much more than choosing pretty fonts, it’s a very important components in UI design, Good typography establishes a strong, cohesive visual hierarchy that helps to present content clearly and efficiently to users.
+
+<br />
+
+## Typeface
+Forma strives for a performant, clean experience to our users, hence, 
+we chose our font stack to match the system fonts that comes preloaded with the OS on your device.
+
+These are the font family we use in our systems...
+
+![Typefaces demonstration, San Francisco, Lucida Grande, Neue Helvetica, Arial, and Segoe UI](./assets/typeface.png)
+
+Our font family consists of different font types, they are known as *system fonts*, 
+every operating system comes with at least one default font...
+
+**San Francisco** appears on Safari (Mac OS X and iOS); **Neue Helvetica** and **Lucida Grande** appears on older versions of Mac OS X; 
+**Segoe UI** targets Windows and Windows phone; **Arial** is available on almost all operating systems.
+
```

Comment:
The Forma 36 font family is made up of _system fonts_. Every operating system comes with at least one system font.